### PR TITLE
Generate periodic statistics at client

### PR DESCRIPTION
**Pull request by dominiklohmann**

### CODE REVIEW

**Comment by dominiklohmann** [comment]
[story/ch9335]

File: scripts/vast-du [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env sh
```
Comment (mavam) [inline scripts/vast-du]: I think a shebang of `#!/bin/sh` is also POSIX compliant.

File: scripts/vast-du [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Usage examples:
+#   vast-du -N import zeek -r libvast_test/artifacts/logs/zeek/conn.log
+#	vast-du -N export ascii '#type ~ /zeek.*/'
+
+set -e
+
+if ! command -v vast >/dev/null 2>&1; then
+	echo "vast not found"
+	exit
```
Comment (mavam) [inline scripts/vast-du]:
```suggestion
	exit 1
```

**Comment by dominiklohmann** [comment]
I’ve addressed the feedback. The script isn’t quite up to the usual standard, because it’s just what I used for testing locally and we decided it was worth keeping.

**Pull request merged**